checkov/terraform/context_parsers/parsers/resource_context_parser.py

```diff
@@ -3,11 +3,18 @@
 
 class ResourceContextParser(BaseContextParser):
     def __init__(self):
-        definition_type = 'resource'
+        definition_type = "resource"
         super().__init__(definition_type=definition_type)
 
-    def get_block_type(self):
-        return self.definition_type
+    def _is_block_signature(self, line_tokens, entity_context_path):
```

Comment:
"_" prefix is for private methods. but you call this method outside of this class - maybe that should be changed?

Reply:
@schosterbarak Thanks, it's a code leftover. the implementation is in `base_context_parser` and usage is only class-internal.